shared/constants/index.js

```diff
@@ -56,6 +56,166 @@ export const SITE_ROUTING = {
   "Landing Page": "/lp",
 };
 
+
+export const JMS__TLC_COLORS = [
```

Comment:
new color token constants added. Should be kept in sync with the tokens in the app when possible